.circleci/config.yml

```diff
@@ -69,60 +72,155 @@ commands:
           command: |
             conda create -n hydra python=<< parameters.py_version >> pywin32 -qy
             conda activate hydra
-            pip install nox dataclasses
+            pip install nox@git+git://github.com/theacodes/nox.git@8dcefbf8f59293bc1925dd13376dd6d3aa66e53a
+            pip install dataclasses
```

Comment:
why do we need to install dataclasses here?

Reply:
this was copy pasted from previous versions of the file. `dataclasses` is needed for python 3.6 to work.

Reply:
needed for what? Doesn't pip installing hydra bring it if it's needed (aka on 3.6)?
try to remove it and see if it works.

Reply:
Yea i tried to remove it, but that caused the tests to fail.

Reply:
curious to know why.
Hydra is installing it's own dependencies inside nox.